ui/components/component-library/button/README.mdx

```diff
@@ -1,20 +1,30 @@
 import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
 import { Button } from './button';
+import { ButtonBase } from '../button-base';
+import { ButtonLink } from '../button-link';
 
 # Button
 
-The `Button` is used for user actions it unifies `ButtonPrimary`, `ButtonSecondary` and `ButtonLink`
+The `Button` is used for user actions. It unifies `ButtonPrimary`, `ButtonSecondary` and `ButtonLink`.
 
 <Canvas>
   <Story id="ui-components-component-library-button-button-stories-js--default-story" />
 </Canvas>
 
 ## Props
 
-The `Button` accepts all props below as well as all [ButtonPrimary](/ui-components-component-library-button-primary-button-primary-stories-js--default-story), [ButtonSecondary](/ui-components-component-library-button-secondary-button-secondary-stories-js--default-story), [ButtonLink](/ui-components-component-library-button-link-button-link-stories-js--default-story), and [Box](/docs/ui-components-ui-box-box-stories-js--default-story#props) component props
+The `Button` accepts all props below as well as [Box](/docs/ui-components-ui-box-box-stories-js--default-story#props) component props
 
 <ArgsTable of={Button} />
 
+The `Button` accepts all [ButtonBase](/docs/ui-components-component-library-button-base-button-base-stories-js--default-story) component props
+
+<ArgsTable of={ButtonBase} />
+
+The `Button` accepts all [ButtonPrimary](/docs/ui-components-component-library-button-primary-button-primary-stories-js--default-story), [ButtonSecondary](/docs/ui-components-component-library-button-secondary-button-secondary-stories-js--default-story), and [ButtonLink](/docs/ui-components-component-library-button-link-button-link-stories-js--default-story) component props
+
+<ArgsTable of={ButtonLink} />
+
 ### Type
```

Comment:
Nit: Update the jsx for `Type`
```suggestion
import { Button, BUTTON_TYPES } from '../ui/component-library';
```

Reply:
Will need to add `BUTTON_SIZES` and `BUTTON_TYPES`  to the component-library index file